api/v1/endpoints/queries.sql

```diff
@@ -83,14 +83,18 @@ WITH object_data AS (
     SELECT
         a.id AS assessment_id,
         a.tid AS assessment_name,
-        a.title as assessment_title,
+        a.title AS assessment_title,
```

Comment:
For consistency with the `select_assessment_instances` query, can we add
`assessment_label`, `assessment_set_abbreviation`, and `assessment_number`?

Reply:
Along these lines, should I add assessment_title to select_assessment_instances?  (Or remove it from select_assessment_access_rules?)

Reply:
Anticipating/hoping for that response, I already put it in there!

Reply:
Haha, nice! 😄